Alacritty changed configuration file format from YML to TOML

huuxfluux opened the pull request:
new file:   alacritty/alacritty.toml
	deleted:    alacritty/alacritty.yml


Updated to .TOML and removed 'custom_cursor_colors: true' since it gave error.

Commit:
Changes to be committed:
	new file:   alacritty/alacritty.toml
	deleted:    alacritty/alacritty.yml

Updated to .TOML and removed 'custom_cursor_colors: true' since it gave error.

fenetikm commented:
Thanks for that - probably want to keep the old version for those on older versions of alacritty - could you please change your PR? @huuxfluux

huuxfluux committed:
Kept legacy YML file

Kept for users with older versions. added comment that alaccritty changed to TOML with v0.13,

huuxfluux commented:
Alright!

fenetikm commented:
Awesome @huuxfluux - thanks so much.
